lm_eval/models/huggingface.py

```diff
@@ -1026,7 +1034,7 @@ def _collate(x):
                 self._model_call(batched_inps, **call_kwargs), dim=-1
             )  # [batch, padding_length (inp or cont), vocab]
 
-            for (cache_key, _, _), logits, inplen, cont_toks in zip(
+            for (request_str, cxt_tokens, _), logits, inplen, cont_toks in zip(
```

Comment:
typo cxt_tokens -> ctx_tokens?

```suggestion
            for (request_str, ctx_tokens, _), logits, inplen, cont_toks in zip(
```